css/style.css

```diff
@@ -17,3 +17,64 @@ H2: Font Size = 18px
 	Font-Family: cursive, Helvetica, sans-serif;
 
 */
+
+body {
+  width: 984px;
+  margin: 0 auto;
+  font-size: 14px;
+  font-family: "Times New Roman", serif;
+  background-color: white;
+  background-image: url("../images/pattern1.png");
+}
+h1 {
+  font-size: 50px;
+  font-family: arial, helvetica, sans-serif;
+  text-transform: uppercase;
+  color: #FFFFFF;
+  background-color: #EE7871;
+  padding: 5px 0 0 10px;
+}
+h2 {
+  font-size: 18px;
+  text-transform: uppercase;
+  font-family: cursive, helvetica, sans-serif;
+  background-color: #EE7871;
+  padding: 5px 0 5px 10px;
+}
+h3 {
+  font-size: 18px;
+  text-decoration: underline;
+  text-transform: uppercase;
+  font-family: cursive, helvetica, sans-serif;
+  margin: 5px 0 0 10px;
+}
```

Comment:
most of the h3 styles are the same as h2. You can combine by comma separating the selectors:

```css
h2, h3{
  font-size: 18px;
}
```